### ISSUE TITLE
Allow scrolling mods list during "Download Mods", or, indicate when a mod already is downloaded

### ISSUE THREAD

**Issue by Xetaxheb** [PrismLauncher/PrismLauncher#848]
### Role

Installing mods

### Suggestion

Either unlock the ui for scrolling mousewheel over mods list to see what is on the list or add an indicator on the list like the underlining when a mod is already going to be downloaded

### Benefit

It's impossible to see if you already have a mod downloaded making it redundant to download again or check version unless it's coincidentally visible

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Basically this (Fabric API) but for when an existing mod jar matches the auto-recommended download...
![image](https://user-images.githubusercontent.com/3267055/218285149-bca0bf16-deb7-4680-b902-80768c797946.png)
I assume versions are polled when a mod is selected in the list? This is mostly meaning some indicator that "you are downloading a duplicate jar if you click select mod for download". I don't know if the jar info is already polled or if this would require extra api calls, if the latter it's probably not worth it.

**Comment by Trial97** [comment]
The latest 7.1 release allows you to view installed mods(if they have metadata and the provider matches).

**Comment by Xetaxheb** [comment]
duplicate of #447, resolved by #1135 in 7.1
although being able to scroll the mods list during open download mods ui would be nice, probably either not possible with QT or just not worth the effort with mods being highlighted now.

anyone else coming across this keep in mind you can generate metadata for 99% of your mods by doing "check for updates" even if you cancel the update process at the checklist
